quay-enterprise/build-support.md

```diff
@@ -45,15 +45,15 @@ Use the environment variable `SERVER` to tell the worker the hostname at which Q
 Here's what the full command looks like:
 
 ```sh
-docker run --restart on-failure -e SERVER=wss://myquayenterprise -v /var/run/docker.sock:/var/run/docker.sock quay.io/coreos/quay-builder:v2.3.4
+docker run -it -e SERVER=wss://myquayenterprise -v /var/run/docker.sock:/var/run/docker.sock --entrypoint /bin/sh quay.io/coreos/quay-builder:v2.3.4
```

Comment:
I still don't know why we need this change. All of the ticket comments had to do with the syntax for overriding the entrypoint. In this case we're not changing the entrypoint. In fact, this will just run bash, and will most certainly NOT run the quay builder. Changing it to `-it` makes it interactive so ctrl+c will end the process. That's only needed if you're running it from an interactive terminal. This generally applies to the command below as well.